core/src/main/java/com/expedia/adaptivealerting/core/evaluator/RmseEvaluator.java

```diff
@@ -0,0 +1,26 @@
+package com.expedia.adaptivealerting.core.evaluator;
+
+import com.expedia.adaptivealerting.core.util.AssertUtil;
+import com.expedia.adaptivealerting.core.util.MathUtil;
+
+/**
+ * Calculates Root-mean-squared error i.e. differences between values (sample and prediction values) predicted by a
+ * model.
+ * 
+ * @author kashah
+ *
+ */
+
+public class RmseEvaluator implements Evaluator {
+    @Override
+    public double getScore(double[] actual, double[] prediction) {
+        AssertUtil.isTrue(actual.length == prediction.length, "actual.length != prediction.length");
+        int n = actual.length;
+        double resSumSquares = 0.0;
+        for (int i = 0; i < n; i++) {
+            resSumSquares += MathUtil.getSquare(actual[i] - prediction[i]);
+        }
+        return MathUtil.roundToThree(MathUtil.getSquareRoot((resSumSquares / n)));
```

Comment:
This doesn't actually round the double to three digits (despite appearances)--see below. Also, is there any point to rounding to three here? I don't see it, and it just increases the cost of the calculation.